heron/spi/src/java/com/twitter/heron/spi/scheduler/IScalable.java

```diff
@@ -30,8 +31,10 @@
    * Requests new containers for scaling a topology
    *
    * @param containersToAdd Set of containers to be added by the scheduler
+   * @return Map from container-id assigned by packing algorithm, to the actual deployed container-id
```

Comment:
- the actual deployed container-id is a ContainerPlan object? the comment is a little confusing.
- if the mapping is not needed in a specific scheduler, the function should return empty map?

suggest to have a comment about it.

Reply:
updated